lldb/source/API/SBProcess.cpp

```diff
@@ -1255,6 +1255,95 @@ lldb::SBFileSpec SBProcess::GetCoreFile() {
   return SBFileSpec(core_file);
 }
 
+addr_t SBProcess::GetAddressMask(AddressMaskType type,
+                                 AddressMaskRange addr_range) {
+  LLDB_INSTRUMENT_VA(this, type, addr_range);
```

Comment:
`lldb-instr` always puts a newline after `LLDB_INSTRUMENT_VA`. We should do the same here to be consistent with the rest of the SB API.